### PR TITLE
Update openvanilla from 1.4.0 to 1.5.0

### PR DESCRIPTION
**Pull request by CryoTheRenegade**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by CryoRenegade** [comment]
Looks lie the .app is no longer available in the installer app and has been replaced with a terminal executable.

**Comment by CryoRenegade** [comment]
Yea, i have no idea how to reformat the .rb to make the app, help is welcome!

**Comment by vitorgalvao** [comment]
You’ll need to replace `input_method` with an [`installer :manual`](https://github.com/homebrew/homebrew-cask/blob/c129758c8086d86b3d33cc8c2045b4a26bee2a83/doc/cask_language_reference/stanzas/installer.md#installer-manual). It will also require [`uninstall`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md).

File: Casks/openvanilla.rb [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 
   # github.com/openvanilla/openvanilla was verified as official when first introduced to the cask
   url "https://github.com/openvanilla/openvanilla/releases/download/#{version}/OpenVanilla-Installer-Mac-#{version}.zip"
   appcast 'https://github.com/openvanilla/openvanilla/releases.atom'
   name 'OpenVanilla'
   homepage 'https://openvanilla.org/'
 
-  input_method 'OpenVanillaInstaller.app/Contents/Resources/OpenVanilla.app'
+  installer manual:'InstallOpenVanilla.app'
+  uninstall
```
Comment (vitorgalvao) [inline Casks/openvanilla.rb]: This is not a valid `uninstall` stanza. Please see the linked documentation.

Also, as it is, the cask wouldn’t pass `style`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Found a better method: https://github.com/Homebrew/homebrew-cask/pull/74196.

Thank you anyway for your work here.

**Pull request closed without merging**